cs/create-cs.ftd

```diff
@@ -0,0 +1,210 @@
+something like this.
+
+-- ftd.image: $fastn-assets.files.images.cs.how-to-create-cs.5.png
+height.fixed.px: 400
+margin-vertical.px: 10
+
+-- ds.h1: Let's modify the colors
+
+All the colors details of this newly color-scheme repository are located inside
+the `colors.ftd` file.
+
+-- ftd.image: $fastn-assets.files.images.cs.how-to-create-cs.6.png
+height.fixed.px: 400
+margin-vertical.px: 10
+
+-- ds.markdown:
+
+This file holds the colors of this color-scheme repository
+so modifying this file will change this color-scheme repository. The colors.ftd
+file will look something like this at first. You will see a lot of
+pre-defined default colors which are provided by fastn.
+
+-- ftd.image: $fastn-assets.files.images.cs.how-to-create-cs.7.png
+height.fixed.px: 400
+margin-vertical.px: 10
+
+-- ds.markdown:
+
+Ideally, we should change all the colors inside `colors.ftd` file based on
+our requirements of our custom color-scheme. But for demonstration purpose,
+we will modify only text color for now. Let's say we change the text color
+to yellow (#FFFF00) for dark mode.
+
+**Proposed change**: Current text color (for dark mode) -> #FFFF00 (yellow)
+
+To edit this file, click on the edit button
+which you can find at the top-right of this page.
+
+-- ftd.image: $fastn-assets.files.images.cs.how-to-create-cs.8.png
+height.fixed.px: 400
+margin-vertical.px: 10
+
+-- ds.markdown:
+
+After clicking on the edit button, we can make changes to this `colors.ftd` file.
+Navigate to the text color variable and change its dark mode color
+to #FFFF00 (yellow). After doing that commit the changes.
+
+-- ftd.image: $fastn-assets.files.images.cs.how-to-create-cs.9.png
+height.fixed.px: 400
+margin-vertical.px: 10
+
+-- ds.h1: Configure index.ftd
+
+Before we start using our colors, we need to configure some details
+defined inside `index.ftd`.  Initially, the `index.ftd` will look like this.
+
+-- ftd.image: $fastn-assets.files.images.cs.how-to-create-cs.10.png
+height.fixed.px: 400
+margin-vertical.px: 10
+
+-- ds.markdown:
+
+By default, `index.ftd` will render the default fastn color scheme. So we
+need to modify it to show our colors (defined inside colors.ftd)
+from this repository. To do that, we will modify few lines which you can
+easily find out from the pre-commented lines.
+
+-- ftd.image: $fastn-assets.files.images.cs.how-to-create-cs.11.png
+height.fixed.px: 400
+margin-vertical.px: 10
+
+-- ds.markdown:
+
+To modify this file, you will first need to click on the edit button which you can find
+on the top-right. After clicking on the edit button, you can edit this file
+
+-- ftd.image: $fastn-assets.files.images.cs.how-to-create-cs.12.png
+height.fixed.px: 400
+margin-vertical.px: 10
+
+-- ds.markdown:
+
+After modifying index.ftd, we will see the contents like this.
+
+-- ftd.image: $fastn-assets.files.images.cs.how-to-create-cs.13.png
+height.fixed.px: 400
+margin-vertical.px: 10
+
+-- ds.h1: Let's make our repository live
```

Comment:
There is a lot of copy paste of this material. Write it in a component and use it from everywhere.